apps/myaccount/src/store/actions/global.ts

```diff
@@ -88,4 +89,14 @@ export const setSupportedI18nLanguages = (
 ): SetSupportedI18nLanguagesActionInterface => ({
     payload: languages,
     type: GlobalActionTypes.SET_SUPPORTED_I18N_LANGUAGES
+    });
+
+/**
+ * Redux action to set an action to open.
+ *
+ * @param {string} action - The action to set to open.
+ */
+export const setOpenAction = (action: string): SetOpenAction => ({
```

Comment:
Better if we can rethink the name

Reply:
Fixed by 26e3ed87.
`setOpenAction` has been replaced with `setActiveForm`.